datafusion/proto/proto/datafusion.proto

```diff
@@ -413,6 +414,8 @@ message Wildcard {
 message PlaceholderNode {
   string id = 1;
   datafusion_common.ArrowType data_type = 2;
```

Comment:
I recommend adding a  comment here explaining that this doesn't serialize the Field directly to maintain backwards compatibiilty with older version